nginx/conf.yaml.example

```diff
@@ -16,6 +16,11 @@ instances:
   #
   # - nginx_status_url: http://example2.com:1234/nginx_status/
   #   ssl_validation: False
+
+  #   The (optional) skip_proxy parameter will bypass any proxy
+  #   settings enabled and attempt to reach the URL directly.
+  #
+  #   skip_proxy: false
```

Comment:
Shouldn't this be `no_proxy` instead? (based on this: https://github.com/DataDog/dd-agent/blob/5.21.0/checks/__init__.py#L396)